init.lua

```diff
@@ -26,7 +26,7 @@ local basic_properties = {
 		wall_bottom = {-0.4375, -0.5, -0.3125, 0.4375, -0.4375, 0.3125},
 		wall_side   = {-0.5, -0.3125, -0.4375, -0.4375, 0.3125, 0.4375},
 	},
-	groups = {snappy=1}, --2,dig_immediate=2,attached_node=1},
+	groups = {snappy=1, oddly_breakable_by_hand=2, attached_node=1},
```

Comment:
Hmmm, I could not find a good API documentation describing those little details. I think I now see what this is supposed to do. +1